dbt_copilot_helper/templates/addons/env/aurora-postgres.yml

```diff
@@ -45,10 +45,16 @@ Mappings:
       DBMinCapacity: {{ config.min_capacity }}
       DBMaxCapacity: {{ config.max_capacity }}
       DeletionPolicy: {{ config.deletion_policy }}
+{%- if config.snapshot_id %}
+      SnapshotIdentifier: {{ config.snapshot_id }}
+{%- else %}
+      SnapshotIdentifier: ""
+{%- endif %}
```

Comment:
Maybe...
```suggestion
      SnapshotIdentifier: {{ config.snapshot_id if config.snapshot_id else "" }}
```

Reply:
Yes, it is documented in the platform docs. I'm making a few tweaks to it as we speak.

The suggested change doesn't preserve the quotation marks which is needed, but I've committed and tweaked your change slightly in https://github.com/uktrade/copilot-tools/pull/258/commits/83564caceab58e581c5b901dfbe2d08fd187e63c.